src/features/MentorPage/MentorsFilter/Filters/FiltersButton.tsx

```diff
@@ -3,28 +3,33 @@ import {
   selectSearchString,
   selectSelectedSkills,
 } from '../mentorsFilterSlice';
-import { useAppSelector } from '../../../../store';
+import { useAppSelector } from '@/store';
+import { useTranslation } from 'react-i18next';
 
 import styled from 'styled-components';
-import * as cssVariables from '../../../../components/variables';
-import { Text } from '../../../../components/Text/Text';
-import SearchIcon from '../../../../static/icons/search.svg';
+import * as cssVariables from '@/components/variables';
```

Comment:
Now that we change these to alias imports, (i like them better too) at the same time, could you destructure import only what is used?
Personally I prefer it that way than the `* as blabla` =P

Reply:
Yes, I prefer it that way too, but didn't refactor everything yet.

Reply:
Done! Variable imports using * are now gone!

Reply:
Types directory mistake corrected!

Reply:
Broken tests also fixed!